viadot/tasks/sql_server.py

```diff
@@ -127,3 +127,41 @@ def run(
             f"Successfully downloaded {nrows} rows and {ncols} columns of data to a DataFrame."
         )
         return df
+
+
+class SQLServerQuery(Task):
+    def __init__(
+        self,
+        config_key: str = None,
+        *args,
+        **kwargs,
+    ):
+        """
+        Task for running query on SQL Server.
```

Comment:
grammar: should be either "for running queries on SQL Server" or "for querying SQL Server"